### PR TITLE
Create lab2_part3.ino

### PR DESCRIPTION
**Pull request by iwerbomark**

### CODE REVIEW

File: students/markusiwerbo/lab2_part3/lab2_part3.ino [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+void setup() {
+  pinMode(led, OUTPUT);
+  Serial.begin(9600);
+}
+ 
+void loop() {
+  int analog_value = analogRead(input_pin);     // Read analog voltage from the sensor
+ 
+  
+  float distance = analog_value * voltage_per_in;   // Convert analog voltage to inches
```
Comment (adampbeardsley) [inline students/markusiwerbo/lab2_part3/lab2_part3.ino:20]: This does not convert to inches.

File: students/markusiwerbo/lab2_part3/lab2_part3.ino [hunk 2]
```diff
@@ -0,0 +1,36 @@
+//Lab 2 part 3
+//Distance sensor as analog input 
+
+const int input_pin = A0;   //Analog input from sensor
+const int led = 13;         //Digital utput pin
+const float voltage = 5.0;        //Sensor power supply
+const float voltage_per_in = voltage / 512.0; 
+ 
+const float half_inch_ref = 0.5;
```
Comment (adampbeardsley) [inline students/markusiwerbo/lab2_part3/lab2_part3.ino:9]: Is the sensor reliable to this distance?

**Review by adampbeardsley — APPROVED** [review]
Merge when ready. More comments coming on D2L.

File: students/markusiwerbo/lab2_part3/lab2_part3.ino (hunk 2 above)
Comment (iwerbomark) [reply students/markusiwerbo/lab2_part3/lab2_part3.ino:9]: The LED turned on at a larger distance. It stayed on at that distance.

**Pull request merged**